sktime/transformations/series/fourier.py

```diff
@@ -45,12 +45,16 @@ class FourierFeatures(BaseTransformer):
     ----------
     sp_list : List[float and/or str]
         List of seasonal periods. Can be defined with the following options:
+
         * float : Periodicity defined as number of timesteps since the beginning of the
-        data seen in `fit`.
+        data seen in ``fit``.
+
         * string : Periodicity defined as a column name in X that contains the
-        :math:`t/sp` values
+        :math:`t/sp` values.
```

Comment:
Same as above. You can try the suggestion, but it's not tested and may not work.
```suggestion
            :math:`t/sp` values.
```

Reply:
The rst syntax for line blocks seems to fix the problem.

Reply:
I was not aware of that syntax, it's great to know.